spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -310,6 +321,10 @@ public void setConnectionTimeout(Duration connectionTimeout) {
 		this.connectionTimeout = connectionTimeout;
 	}
 
+	public void setConnectionTimeout(int requestedChannelMax) {
```

Comment:
That doesn't sound right.

Reply:
Thank you, it was an honest mistake to make after several hours of late night troubleshooting why is 2047 still limit for number of channels on production even after changing server configuration to unlimited.

Fixed in updated PR, thanks for suggestion.